BM/tdx-compliance/tdx-compliance.c

```diff
@@ -499,6 +595,42 @@ static int __init tdx_tests_init(void)
 	cur_cr4 = get_cr4();
 	pr_buf("cur_cr0: %016llx, cur_cr4: %016llx\n", cur_cr0, cur_cr4);
 
+	unsigned long cr0;
+	unsigned long flags; 
+	store_idt(&idt_descriptor);
```

Comment:
local_irq_save(flags); should be done before store_idt(), to avoid context switch happen during you update the idt()